ext/TrixiMakieExt.jl

```diff
@@ -2,10 +2,7 @@
 module TrixiMakieExt
 
 # Required for visualization code
-# We do not check `isdefined(Base, :get_extension)` since Julia v1.9.0
-# does not load package extensions when their dependency is loaded from
-# the main environment
-if VERSION >= v"1.9.1"
+if isdefined(Base, :get_extension)
```

Comment:
```suggestion
@static if isdefined(Base, :get_extension)
```
Should this be `@static` for performance/precompilation reasons?

**EDIT**: Just saw you also used it in `src/Trixi.jl`, thus consistency would be another argument for this 😎

Reply:
`@static` is not required here, so I do not see any reason to use it here. It will be required around the `@require` block once we move everything to package extensions and only load Requires.jl on old versions of Julia

Reply:
OK.